packages/genesys-spark-components/src/components/stable/gux-icon/icons/custom/genesys.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="M8.34769 11.4789H9.97291C10.5403 11.4789 11.0828 11.6996 11.4817 12.0899C11.8803 12.48 12.1027 13.0074 12.1027 13.5557C12.1027 14.104 11.8803 14.6313 11.4817 15.0215C11.0828 15.4118 10.5403 15.6325 9.97291 15.6325H8.34769C7.7803 15.6325 7.23773 15.4118 6.83888 15.0215C6.44029 14.6313 6.21784 14.104 6.21784 13.5557C6.21784 13.0074 6.44029 12.48 6.83888 12.0899C7.23773 11.6996 7.7803 11.4789 8.34769 11.4789Z"/><path d="M5.68121 5.17031H9.8147C10.12 5.17031 10.4221 5.22927 10.7038 5.34367C10.9855 5.45802 11.241 5.62551 11.4558 5.83614C11.6707 6.04673 11.8407 6.29638 11.9565 6.57064C12.0724 6.84485 12.1319 7.13849 12.1319 7.43487C12.1319 7.73125 12.0724 8.02489 11.9565 8.29911C11.8407 8.57332 11.6707 8.82296 11.4558 9.0336C11.241 9.24424 10.9855 9.41167 10.7038 9.52607C10.4221 9.64047 10.12 9.69939 9.8147 9.69939H5.68121C5.06435 9.69939 4.47417 9.45911 4.04014 9.0336C3.60636 8.60833 3.36401 8.03308 3.36401 7.43487C3.36401 6.83661 3.60636 6.26136 4.04014 5.83614C4.47417 5.41064 5.06435 5.17031 5.68121 5.17031Z"/><path d="M9.07178 1.87754C9.07178 1.04164 9.74938 0.364012 10.5853 0.364012C11.4212 0.364012 12.0988 1.04164 12.0988 1.87754C12.0988 2.71344 11.4212 3.39108 10.5853 3.39108C9.74938 3.39108 9.07178 2.71344 9.07178 1.87754Z"/><path d="M9.97291 11.1149H8.34769C7.68627 11.1149 7.05195 11.372 6.58426 11.8298C6.11658 12.2875 5.85383 12.9084 5.85383 13.5557C5.85383 14.2031 6.11658 14.8239 6.58426 15.2816C7.05195 15.7393 7.68627 15.9965 8.34769 15.9965H9.97291C10.6343 15.9965 11.2686 15.7393 11.7363 15.2816C12.204 14.8239 12.4667 14.2031 12.4667 13.5557C12.4667 12.9084 12.204 12.2875 11.7363 11.8298C11.2686 11.372 10.6343 11.1149 9.97291 11.1149Z"/><path d="M9.8147 4.80631H5.68121C4.97011 4.80631 4.28814 5.08325 3.78531 5.57619C3.28248 6.06913 3 6.73774 3 7.43487C3 8.132 3.28248 8.80056 3.78531 9.29355C4.28814 9.78649 4.97011 10.0634 5.68121 10.0634H9.8147C10.1668 10.0634 10.5155 9.99544 10.8408 9.86334C11.1661 9.73125 11.4617 9.5376 11.7106 9.29355C11.9596 9.04946 12.1571 8.75967 12.2918 8.44075C12.4266 8.12183 12.4959 7.78005 12.4959 7.43487C12.4959 7.08969 12.4266 6.74786 12.2918 6.42894C12.1571 6.11002 11.9596 5.82028 11.7106 5.57619C11.4617 5.33209 11.1661 5.13849 10.8408 5.0064C10.5155 4.87431 10.1668 4.80631 9.8147 4.80631Z"/><path d="M8.70774 1.87754C8.70774 0.840607 9.54835 0 10.5853 0C11.6222 0 12.4628 0.840607 12.4628 1.87754C12.4628 2.91448 11.6222 3.75509 10.5853 3.75509C9.54835 3.75509 8.70774 2.91448 8.70774 1.87754Z"/></svg>
```

Comment:
Not sure if this would be considered a breaking change or not?

Reply:
I think we can make the change but we need to have the old version accessible at `gux-icon/icons/deprecated/custom/genesys.svg'